client/src/main/scala/org/http4s/client/package.scala

```diff
@@ -37,11 +38,24 @@ package object client {
   implicit class WithBodySyntax(val method: Method with PermitsBody) extends AnyVal with EntityRequestGenerator
   implicit class NoBodySyntax(val method: Method with NoBody) extends AnyVal with EmptyRequestGenerator
 
+  // implicit val taskFunctor = new Functor[Task] {
+  //   def map[A, B](fa: Task[A])(f: A => B): Task[B] = fa.map(f)
+  // }
+
+  implicit val taskMonad = new Monad[Task] {
```

Comment:
Cool.  I did not know about cats-fs2.  Does that mean that this is also unnecessary, from my prior PR? Thanks  https://github.com/http4s/http4s/commit/cb980c13bcc66df3228412cdd8625e776585e39d#diff-d7e06cf5727b9aef4cf0104789ec4bb7R76